src/functionaltests/java/com/ericsson/ei/utils/SubscriptionRestAPI.java

```diff
@@ -16,9 +16,21 @@
 
 public class SubscriptionRestAPI {
```

Comment:
since you anyway doing refactoring change the name of this class to HttpExecutor or other more appropriate name since this has nothing to do with subscriptions